src/simpleGit.ts

```diff
@@ -89,7 +89,7 @@ export class SimpleGit extends GitManager {
         if (this.plugin.settings.updateSubmodules === true) {
             return true;
         }
-        const status = await this.git.status((err: any) => this.onError(err));
+        const status = await this.git.status([], (err: any) => this.onError(err));
```

Comment:
This should not be needed, because there is a method definition with just the error callback. `simple-git.d.ts`:
```ts
/**
    * Show the working tree status.
    */
   status(options?: types.TaskOptions, callback?: types.SimpleGitTaskCallback<resp.StatusResult>): Response<resp.StatusResult>;

   status(callback?: types.SimpleGitTaskCallback<resp.StatusResult>): Response<resp.StatusResult>;
```

Reply:
running `npm update` made the errors go away :+1: I have dropped this commit